pkg/cmd/workspace/create.go

```diff
@@ -359,6 +339,12 @@ func waitForDial(tsConn *tsnet.Server, workspaceId string, projectName string, d
 }
 
 func readLog(ws *websocket.Conn, stopLogs *bool) {
+	if internal.Version == "v0.0.0-dev" {
+		err := ws.SetReadDeadline(time.Time{})
+		if err != nil {
+			panic(err)
```

Comment:
Yes. It does seem kind of iffy 😅 
This is because, when you're developing and the server hits a breakpoint, while the websocket connection is active, the websocket will fail because the deadline was reached and the connection did not receive updates for some time. This prevents that in development mode.

We could make this more readable by having a `internal.DevelopmentMode` variable maybe that does the comparison.

Reply:
Ah got it. We can add the development mode variable, but this seems okay as well